lib/simple-dom/document/element.js

```diff
@@ -152,6 +152,33 @@ if(Object.defineProperty) {
 			this.__node._setAttribute("style", val);
 		}
 	});
+  
+	Object.defineProperty(Element.prototype, "innerHTML", {
+		get: function() {
+			var html = "";
+			var cur = this.firstChild;
+			while (cur) {
+				html += this.ownerDocument.__serializer.serialize(cur);
+				cur = cur.nextSibling;
+			}
+			return html;
+		},
+		set: function(html) {
+			this.lastChild = this.firstChild = null;
+			var fragment = this.ownerDocument.__parser.parse(html);
+			this.appendChild(fragment);
```

Comment:
I fixed a bug here recently when this was implemented in VDOM - the bug tat is now back in our project. `script` and `style` tags should not be parsed as their content is CDATA. Here it is for reference (the STYLE tag should also get the same behavior): https://github.com/canjs/canjs/blob/v2.3.8/util/vdom/vdom.js#L23